spec/views/hyrax/admin/collection_types/_form_settings.html.erb_spec.rb

```diff
@@ -8,24 +21,47 @@
     end
   end
 
-  before do
-    assign(:form, collection_type_form)
-    allow(view).to receive(:f).and_return(form)
-    render
-  end
+  context "collection_type.collections? is false" do
+    before do
+      collection_type_form.collection_type = collection_type
+      assign(:form, collection_type_form)
+      allow(view).to receive(:f).and_return(form)
+      render
+    end
+
+    it "renders the intructions and warning" do
+      expect(rendered).to match(I18n.t("hyrax.admin.collection_types.form_settings.instructions"))
+      expect(rendered).to match(I18n.t("hyrax.admin.collection_types.form_settings.warning"))
+    end
+
+    INPUT_IDS.each do |id|
+      it "renders the #{id} checkbox to be disabled" do
+        match = rendered.match(/(<input.*id="#{id}".*>)/)
+        expect(match).not_to be_nil
+        next if match.nil?
+        expect(match[1].index('disabled="disabled"')).to be_nil
+      end
+    end
 
-  it "renders the intructions and warning" do
-    expect(rendered).to have_content(I18n.t("hyrax.admin.collection_types.form_settings.instructions"))
-    expect(rendered).to have_content(I18n.t("hyrax.admin.collection_types.form_settings.warning"))
   end
 
-  it "renders the checkboxes" do
-    expect(rendered).to have_selector("input#collection_type_nestable")
-    expect(rendered).to have_selector("input#collection_type_discoverable")
-    expect(rendered).to have_selector("input#collection_type_sharable")
-    expect(rendered).to have_selector("input#collection_type_require_membership")
-    expect(rendered).to have_selector("input#collection_type_allow_multiple_membership")
-    expect(rendered).to have_selector("input#collection_type_assigns_workflow")
-    expect(rendered).to have_selector("input#collection_type_assigns_visibility")
+  context "collection_type.collections? is true" do
```

Comment:
Would you consider renaming the `context` to `when collection_type.collections? is true`? The rspec output reads more idiomatically when `context` blocks begin with e.g., `when` or `with`.